.eslintrc.json

```
@@ -0,0 +1,29 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "jest/globals": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "overrides": [
+        {
+            "files": ["/*.cy.js", "/*.test.ts"],
```

```suggestion
            "files": ["/*.cy.js", "/*.test.js"],
```

Is the `.ts` extension intentional?